### PR TITLE
Filter tags asso

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

**Review by BastienSaulnier — DISMISSED** [review]
SnE

File: frontend/src/pages/Causes.jsx [hunk 1]
```diff
@@ -3,10 +3,12 @@ import "./styles/Causes.scss";
 import FilterTab from "../components/FilterTab";
 import ProfileCard from "../components/ProfileCard";
 import axios from "axios";
+import { useSelector } from "react-redux";
 const { apiCall } = require("../conf");
 
 function Causes() {
   const [associations, setAssociations] = useState([]);
+  const filter = useSelector(state => state.filter);
```
Comment (LoicBrassart) [inline frontend/src/pages/Causes.jsx:11]: Bad Idea

**Review by LoicBrassart — DISMISSED** [review]
I may be wrong, but I think I just read the same code on two different branches, which seems to indicate you created a branch from somewhere other than `dev`. That's not a good idea, as I told you already, and could make me angry enough to be harsher on future PRs -_-

File: frontend/src/pages/Causes.jsx (hunk 1 above)
Comment (BastienSaulnier) [reply frontend/src/pages/Causes.jsx:11]: Will fix this in a future PR, and those two branches where respectively made from dev, no worries :) the code is just the same tbh

**Review by BastienSaulnier — DISMISSED** [review]
SnE

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by BastienSaulnier — APPROVED** [review]
SnE

**Pull request closed without merging**